tests/reducers/navigation/index.test.ts

```diff
@@ -0,0 +1,9 @@
+import { navigationReducer as reducer } from 'src/reducers/navigation/'
+import { navigate } from 'src/actions/navigation/'
+
+describe('navigation reducer', ()=> {
+  it('should initialize correctly', () => {
+  const test = reducer(undefined, { type: '@INIT' })
+  expect(test.routes[0].routeName).toBe('Landing')
```

Comment:
So this test will break as soon as we change the sequence - I would check here for inclusion of specific route in `routes` array :)

Reply:
Breaking test is good-- this should always be set to Landing (at least in a PR), because we don't want to have the navigator initialise to any other screen.